packages/core/src/checkout/checkout-store-status-selector.ts

```diff
@@ -102,6 +102,14 @@ export default interface CheckoutStoreStatusSelector {
      */
     isLoadingPaymentMethod(methodId?: string): boolean;
 
+    /**
+     * Checks whether wallet buttons are loading.
+     *
+     * @param methodIds - The identifiers of the payment methods to check.
+     * @returns True if any provided wallet button method is loading, otherwise false.
+     */
+    isLoadingWalletButtons(methodIds?: string[]): boolean;
```

Comment:
```suggestion
    isLoadingWalletButton(methodId: string[]): boolean;
```
To be consistent with other methods that accept a singular method ID?

Reply:
At the end of loading wallet buttons in `checkout-js`, I want to execute something like this.
```
const isLoading = isLoadingWalletButtons(methodIds);

useEffect(() => {
        if(!isLoading){
            setMethodIds(getInitializedWalletButtons(methodIds) ?? []);
        }
    }, [isLoading]);
```
Therefore, I need a way to check all buttons' loading status together.

I just wanted to confirm that this approach in `checkout-js` is good and that we are only renaming here.

Reply:
@davidchin What about this one? 😝

Reply:
Could you filter out the IDs that are not initialized? e.g.:

```ts
const isLoading = methodIds.filter(isLoadingWalletButton).length > 0;
```

Reply:
Hi DC, based on your feedback, I have updated the code, and `checkout-js` implementation looks like below.
```
    const isLoading = methodIds.filter((methodId) => isLoadingWalletButton(methodId)).length > 0;

    useEffect(() => {
        if(!isLoading){
            const initializedMethodIds = methodIds.filter((methodId) => isInitializedWalletButton(methodId));

            setMethodIds(initializedMethodIds);
        }
    }, [isLoading]);
```

Reply:
Hi @davidchin , based on your feedback, I have updated the code, and `checkout-js` implementation looks like below.
```
    const isLoading = methodIds.filter((methodId) => isLoadingWalletButton(methodId)).length > 0;

    useEffect(() => {
        if(!isLoading){
            const initializedMethodIds = methodIds.filter((methodId) => isInitializedWalletButton(methodId));

            setMethodIds(initializedMethodIds);
        }
    }, [isLoading]);
```